pkg/hostagent/snats.go

```diff
@@ -353,7 +353,8 @@ func (agent *HostAgent) handleSnatUpdate(policy *snatpolicy.SnatPolicy) {
 		for _, service := range services {
 			uids, _ := agent.getPodsMatchingObject(service, policy.ObjectMeta.Name)
 			poduids = append(poduids, uids...)
-			key, err := cache.MetaNamespaceKeyFunc(service)
+			// key, err := cache.MetaNamespaceKeyFunc(service)
```

Comment:
pl remove this

Reply:
Will address above comment & removed the commented code.